product-service/src/functions/getProductById/handler.ts

```diff
@@ -0,0 +1,32 @@
+import 'source-map-support/register';
+import { APIGatewayProxyEvent } from 'aws-lambda';
+import { FormatJSONResponseType, Sneaker } from '@app/types/types';
+import { formatJSONResponse } from '@libs/apiGateway';
+import { middyfy } from '@libs/lambda';
+import { getAllProducts } from '@libs/getAllProducts';
+import { find } from '@libs/find';
+
+export const handler = async (event: APIGatewayProxyEvent): Promise<FormatJSONResponseType> => {
+  try {
+    const { id } = event.pathParameters;
+    const sneakers: Sneaker[] = await getAllProducts();
+    const product: Sneaker = find(sneakers, id);
+    
+    return formatJSONResponse({
+      statusCode: product ? 200 : 404,
```

Comment:
It is better to avoid magic numbers in the code and in this case it would be great to use enum from constants
```
export enum STATUS {
  SUCCESS = 200,
  NOT_FOUND = 404,
  SERV_ERR = 500
}
```